modules/rgbd/CMakeLists.txt

```diff
@@ -1,15 +1,7 @@
 set(the_description "RGBD algorithms")
 
-find_package(Ceres QUIET)
```

Comment:
Does it make sense to keep both implementations (to compare results) for some timeframe (e.g., until we have robust tests)?